Sources/StreamChat/APIClient/APIClient.swift

```diff
@@ -4,6 +4,13 @@
 
 import Foundation
 
+public struct RetryOptions {
```

Comment:
We don't need this to be public?
```suggestion
struct RetryOptions {
```

Reply:
pretty sure this will be used in other places as well and that customers will want to be able to provide their own retry policy (ie. how many times we retry to send a message before giving up)

Reply:
In that case, we need to create the `public init` or that won't be exposed

Reply:
I would also not expose it unless we have a strong reason/a customer asking for it. Currently, there's no way atm for a customer to customize that but it does increase the public-facing surface and which makes us less flexible in terms of internal design

Reply:
@nuno-vieira 
> In that case, we need to create the public init or that won't be exposed

Are you sure about this?

Reply:
Yes. This is easy to test, just try to create the struct on the demo app. It will say the init is not accessible